src/main/java/org/broadinstitute/hellbender/engine/filters/LibraryReadFilter.java

```diff
@@ -16,7 +16,7 @@
 public final class LibraryReadFilter extends ReadFilter implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName = "library", shortName = "library", doc="The name of the library to keep", optional=false)
+    @Argument(fullName = "library", shortName = "library", doc="The name of the library to keep.", optional=false)
```

Comment:
Currently only one library could be kept (the argument is a single `String`). If multiple libraries could be provided, this will be a collection and Barclay will handle that documentation. There is no exclude option yet.

I opened a ticket (#3144) for addressing this in a different PR to do not include this changes in a doc-related one.